app/src/main/res/layout/fragment_about.xml

```diff
@@ -24,37 +24,18 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
 
-        <LinearLayout
+        <TextView
+            android:id="@+id/about_tos_content"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginTop="24dp"
-            android:background="?selectableItemBackground"
+            android:layout_marginTop="16dp"
             android:onClick="@{ () -> vm.tosLinkClicked() }"
-            android:orientation="horizontal"
-            android:paddingTop="16dp"
-            android:paddingBottom="16dp"
+            android:text="@string/about_tos_content"
```

Comment:
raději bych to změnil jen na 
`tools:text="@string/about_tos_content"`

dokud se nezavolá setString v kódu je lepší nezobrazit nic než html user-unfriendly text

Reply:
takto bysme to měli změnit i pro html ve WelocmeFramgnet a ActivationFragment kde se děje to samé